### PR TITLE
multi: Use installer.type for `python` and `mro`

### PR DESCRIPTION
**Pull request by niheaven**
Test manifests for https://github.com/lukesampson/scoop/pull/3502

Could be merged after above PR merged.

### CODE REVIEW

File: bucket/python.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
     "homepage": "https://www.python.org/",
-    "description": "A programming language that lets you work quickly and integrate systems more effectively.",
```
Comment (rasa) [inline bucket/python.json]: Please add the description back.

**Comment by rasa** [comment]
I tried installing mro locally, but received this error:
```
Running installer script...
DEBUG[1560887074.93424] $out = -------------------------------------------------------------------------------
   ROBOCOPY     ::     Robust File Copy for Windows
-------------------------------------------------------------------------------

  Started : Tuesday, June 18, 2019 7:44:34 PM
   Source : C:\scoop\apps\mro\3.5.3\Microsoft\MRO-3.5.3.0\
     Dest : C:\scoop\apps\mro\3.5.3\

    Files : *.*

  Options : *.* /S /E /DCOPY:DA /COPY:DAT /MOVE /R:1000000 /W:30

------------------------------------------------------------------------------

2019/06/18 19:44:34 ERROR 3 (0x00000003) Accessing Source Directory C:\scoop\apps\mro\3.5.3\Microsoft\MRO-3.5.3.0\
The system cannot find the path specified. -> C:\scoop\apps\scoop\current\lib\core.ps1:422:9
Could not find 'MRO-3.5.3.0'! (error 16)
At C:\scoop\apps\scoop\current\lib\core.ps1:423 char:9
+         throw "Could not find '$(fname $from)'! (error $($proc.ExitCo ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find 'MRO-3.5.3.0'! (error 16):String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find 'MRO-3.5.3.0'! (error 16)

**Comment by niheaven** [comment]
Works fine for me... Have you pull recent commits from https://github.com/lukesampson/scoop/pull/3502?

<details>
<summary>Console Output</summary>

```powershell
λ  scoop update mro -f
mro: 3.5.3 -> 3.5.3
Updating one outdated app:
Updating 'mro' (3.5.3 -> 3.5.3)
Downloading new version
Loading microsoft-r-open-3.5.3.exe from cache
Checking hash of microsoft-r-open-3.5.3.exe ... ok.
Uninstalling 'mro' (3.5.3)
Removing shim for 'r'.
Removing shim for 'rterm'.
Removing shim for 'rscript'.
Unlinking D:\Scoop\apps\mro\current
Installing 'mro' (3.5.3) [64bit]
Loading microsoft-r-open-3.5.3.exe from cache
Extracting microsoft-r-open-3.5.3.exe ... done.
Running installer script...done.
Linking D:\Scoop\apps\mro\current => D:\Scoop\apps\mro\3.5.3
Creating shim for 'r'.
Creating shim for 'rterm'.
Creating shim for 'rscript'.
'mro' (3.5.3) was installed successfully!
Notes
-----
You'll need to type 'r.ps1' or 'r.cmd' to run MRO, because in Powershell 'r' runs the last command. Alternatively 'rterm' can be used to start the
interactive R terminal session.

You can remove Powershell's 'r' command with:
rm alias:\r

... but this only affects your current session: if you'd like to remove it for all future sessions you need to add the command above to your Powershell
profile.

Annoying, right?! You might want to check out Pshazz (scoop install pshazz)--this has a plugin to remove some crazy aliases from Powershell, as well as many
other improvements.
```

</details>

**Comment by rasa** [comment]
Sorry, I didn’t test against https://github.com/lukesampson/scoop/pull/3502 .

**Comment by HUMORCE** [comment]
Reopen when the Core is ready.

**Pull request closed without merging**